apps/translationstudio/next.config.js

```diff
@@ -0,0 +1,6 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
+};
```

Comment:
Could you please update the Next Config to enable these additional settings?

These settings will adjust `npm run build` to generate the build output files in the `./out` directory as static html (index.html, 404.html, etc). This is necessary so that we can upload the bundled app to our deployment server so the app can be served from `https://(.*).ctfcloud.net`

```suggestion
const nextConfig = {
  reactStrictMode: true,
  basePath: '',
  assetPrefix: './',
  output: 'export',
  trailingSlash: true,
  images: {
    unoptimized: true,
  },
};
```